### DISCUSSION TITLE
specs config conflicts with --spec

### DISCUSSION THREAD

**Discussion by anupamtomar** [webdriverio/webdriverio#10174] [Q&A] [answered]
```
specs: [
  './test/specs/**/*.js'
],
```
Does this above configuration conflict with the --spec option if we try to specify a single file execution simultaneously?

I have tried executing a single spec using the --spec argument with the above config but it is executing all the specs, I thought it will overwrite this config but seems it couldn't.

**Comment by christian-bromann** [comment]
> I have tried executing a single spec using the --spec argument with the above config but it is executing all the specs, I thought it will overwrite this config but seems it couldn't.

I does, how did you call the wdio command?

**Reply by anupamtomar** [reply]
```
//package.json

"scripts": {
    "test": "wdio ./wdio.conf.js"
  }
```


`$ TEST_ENV=dev npm run test --spec ./test/specs/myTest.js`

**Reply by christian-bromann** [reply, marked as answer]
you have to do:
```
$ TEST_ENV=dev npm run test -- --spec ./test/specs/myTest.js
```
